src/lib/panels/feedback.ts

```diff
@@ -0,0 +1,297 @@
+   */
+  public dispose() {
+    FeedbackPanel.currentPanel = undefined;
+
+    // Dispose of the current webview panel
+    this._panel.dispose();
+
+    // Dispose of all disposables (i.e. commands) for the current webview panel
+    while (this._disposables.length) {
+      const disposable = this._disposables.pop();
+      if (disposable) {
+        disposable.dispose();
+      }
+    }
+  }
+
+  /**
+   * Defines and returns the HTML that should be rendered within the webview panel.
+   *
+   * @remarks This is also the place where references to CSS and JavaScript files/packages
+   * (such as the Webview UI Toolkit) are created and inserted into the webview HTML.
+   *
+   * @param webview A reference to the extension webview
+   * @param extensionUri The URI of the directory containing the extension
+   * @returns A template string literal containing the HTML that should be
+   * rendered within the webview panel
+   */
+  private _getWebviewContent(webview: Webview, extensionUri: Uri) {
+    const mainUri = getUri(webview, extensionUri, [
+      'webview-ui',
+      'js',
+      'feedback-form.js',
+    ]);
+    const stylesheetUri = getUri(webview, extensionUri, [
+      'webview-ui',
+      'css',
+      'styles.css',
+    ]);
+    const userIdentificationInformation = getUserIdentificationInformation();
+
+    trackEvent(TRACKED_EVENTS.FEEDBACK.FEEDBACK_PANEL_OPENED);
+
+    // Tip: Install the es6-string-html VS Code extension to enable code highlighting below
+    return /*html*/ `
+      <!DOCTYPE html>
+      <html lang="en">
+        <head>
+          <meta charset="UTF-8">
+          <meta name="viewport" content="width=device-width, initial-scale=1.0">
+          <link rel="stylesheet" href="${stylesheetUri}" />
+          <script type="module" src="${mainUri}"></script>
+          <title>HubSpot VSCode Extension Feedback</title>
+        </head>
+        <body>
+          <h1>HubSpot VSCode Extension Feedback</h1>
+          <div class="description">
+            <h3>
+              Thank you for taking the time to provide feedback on the HubSpot VSCode Extension.
+            </h3>
+          </div>
+          <form id="feedback-form" enctype="multipart/form-data">
+            <div class="form-field hidden">
+                <label>Visual Studio Code Version</label><br />
+                <input name="vs-code-version" type="text" value="${userIdentificationInformation.vscodeVersion}" disabled>
+            </div>
+            <div class="form-field hidden">
+                <label>HubSpot Extension Version</label><br />
+                <input name="hubspot-extension-version" type="text" value="${userIdentificationInformation.version}" disabled>
+            </div>
+            <div class="form-field hidden">
+                <label>Machine ID</label><br />
+                <input name="machine-id" type="hidden" value="${userIdentificationInformation.machineId}" disabled>
```

Comment:
does this need to be disabled?

Reply:
also wondering if we could use `readonly` instead of `disabled`, I think readonly still submits

Reply:
Good call, this works much better since I don't need the logic for enabling the disabled inputs.